pkgs/by-name/dh/dhcpig/package.nix

```
@@ -0,0 +1,38 @@
+{ lib
+, python3
+, fetchFromGitHub
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "dhcpig";
+  version = "1.6";
+
+  src = fetchFromGitHub {
+    owner = "kamorin";
+    repo = "DHCPig";
+    rev = version;
```

Helps to avoid branch/tag confusion.

```suggestion
    rev = "refs/tags/${version}";
```